docs/user/get-started.md

```diff
@@ -0,0 +1,333 @@
+# Get Started
+
+For the one who is interested in Vald, this article will show you a way to deploy and run Vald on k8s.
+This article uses Scylla DB as the backend data store for metadata-management and backup-manager.
+Fashion-mnist is used as an example of a dataset.
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is a distributed highly scalable and fast approximate nearest neighbor dense vector search engine.<br>
+It provides the search result of any input (e.g. word, sentence, image and etc.) which is coverted to a multi-dimensional vector when searching the neighbor.<br>
+Vald is designed base on Cloud Native.
+It uses the fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) to search neighbors.
+(If you are interested in ANN benchmarks, please refer to [the official website](http://ann-benchmarks.com/).)
+
+### Main Features
+
+- Auto Indexing
+    - Normally, when changing the Graph Index, the Graph must be locked, but Vald uses distributed index graph, it is extremely difficult for the user to reconstruct the Graph.
```

Comment:
distributed index graph is not related to lock... there are 2 different feature?

Reply:
@kevindiu we definitely  need lock for graph structure in multithreaded  environment.
It's difficult point to dynamically  reconstruct  large scale graph structure

Reply:
maybe I do not understand why we need to write this sentence here?? @@

Reply:
In previous version of vector search API such as NGTD we need stop entire services when reconstructing the index, but due to changes in locking logic and multi-thread management, Vald will not stop the service when reconstructing.

Reply:
it's kind of strong feature, we should describe about it

Reply:
maybe it is better to write it on another main feature something like `No downtime during indexing` @@

Reply:
I will add `Asynchronies indexing` section @@